androidApp/src/main/java/com/mbta/tid/mbta_app/android/stopDetails/TripStopRow.kt

```diff
@@ -190,6 +220,37 @@ fun TripStopRow(
                 }
             }
         }
+        if (disruption != null) {
+            Box(Modifier.height(IntrinsicSize.Min)) {
+                // Trying to get this spacing to look right in Android Studio previews at any device
+                // DPI requires both layers of padding; moving all 40 DP to one side or the other
+                // makes things stop lining up properly. Causality is a superstition.
+                Row(Modifier.padding(start = 6.dp).height(IntrinsicSize.Min).matchParentSize()) {
+                    Column(
+                        Modifier.fillMaxHeight().padding(start = 34.dp).width(20.dp),
+                        horizontalAlignment = Alignment.CenterHorizontally
+                    ) {
+                        ColoredRouteLine(routeAccents.color, Modifier.weight(1f), stateAfter)
```

Comment:
I thought to myself “wait, 48.5 DP doesn’t make sense for like anything, but that was the value that made it all line up in the Android Studio preview” and then I spent forever trying to understand why anything works the way that it does. 48 DP would’ve been the correct value (40 DP of stuff to the left, at the center of a 20 DP column, but the width of the route line itself is 4 DP), but it didn’t line up at the default preview DPI, and the only way to make it look right at every DPI was to have two layers of padding to more closely match the spacing in the row full of content.